hilti/src/rt/types/address.cc

```diff
@@ -61,11 +67,11 @@ Address Address::mask(unsigned int width) const {
     else
         a2 = 0;
 
-    return Address{a1, a2};
+    return Address{a1, a2, _family};
 }
 
 std::variant<struct in_addr, struct in6_addr> Address::asInAddr() const {
-    switch ( family() ) {
+    switch ( (_a1 == 0 && (_a2 & 0xffffffff00000000) == 0) ? AddressFamily::IPv4 : AddressFamily::IPv6 ) {
```

Comment:
Why's this not using family?

Reply:
Good catch, this is a leftover from shuffling things around. Cleaned up.